unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Passport/PlayerPassportHUDView.cs

```diff
@@ -43,8 +55,71 @@ public void SetPassportPanelVisibility(bool visible)
             {
                 mouseCatcher.UnlockCursor();
             }
-            container.SetActive(visible);
             CommonScriptableObjects.playerInfoCardVisibleState.Set(visible);
+
+            animationCancellationToken.Cancel();
+            animationCancellationToken = new CancellationTokenSource();
+
+            if (visible)
+            {
+                container.SetActive(true);
+                ShowPassportAnimation(animationCancellationToken.Token).Forget();
+            }
+            else
+            {
+                playerInfoView.ResetPanelOnClose();
+                HidePassportAnimation(animationCancellationToken.Token);
+            }
+
+        }
+
+        private async UniTaskVoid ShowPassportAnimation(CancellationToken cancellationToken)
+        {
+            cancellationToken.ThrowIfCancellationRequested();
+            passportMask.anchoredPosition = new Vector2(0, -180);
+            passportCanvas.alpha = 0;
+            passportCanvas.DOFade(1, 0.3f)
+                          .SetEase(Ease.Linear);
+            try
+            {
+                Vector2 endPosition = new Vector2(0, 0);
+                Vector2 currentPosition = passportMask.anchoredPosition;
+                DOTween.To(() => currentPosition, x => currentPosition = x, endPosition, 0.3f).SetEase(Ease.OutBack);
+                while (passportMask.anchoredPosition.y < 0)
+                {
+                    passportMask.anchoredPosition = currentPosition;
+                    await UniTask.NextFrame(cancellationToken);
+                }
+            }
+            catch (OperationCanceledException)
+            {
+
+            }
```

Comment:
Although we don't want to do anything special in this case when an exception is thrown, don't we want al least to show some kind of error log informing about it?